docs/igvc2018/EKFTesting.md

```diff
@@ -0,0 +1,46 @@
+# Localisation Testing
+Documents how we tested for the EKF for IGVC 2018
+
+# Installation and Setup (Software)
+
+1. Run 'cd ~/Snowflake && git checkout master'
+2. Run 'cd ~/Snowflake/setup_scripts && ./setup_udev_rules.sh'
+  - If the script fails make sure that ./install_dependencies.sh successfully completed before.
+3. Compile everything by running 'catkin_make'.
+
+# Installation and Setup (Hardware)
+
+1. Plug the encoder and IMU into the USB hub.
+2. Place your laptop on the tower platform.
+3. Plug the USB hub into your laptop.
+4. Plug the GPS into your laptop on a separate port from the USB hub.
+5. Place the GPS antenna at the peak of the tower.
+6. Move the test cart to an area with good GPS signal
+
+# Collecting rosbags for testing
+
+1. Open up 4 terminals (or equivalent).
+2. Run 'roscore' on the first terminal.
+2. Run 'cd ~/Snowflake && source devel/setup.sh' on the second terminal.

```

Comment:
should it be ``source devel/setup.bash`` instead?

Reply:
Depends on which shell you use really, I think `.sh` is the most neutral one. I source `setup.zsh` on [zsh](https://en.wikipedia.org/wiki/Z_shell).